karate-tests/src/test/java/patients/healthcareWorker/updatePatientDetails/oasExamples.feature

```diff
@@ -405,6 +351,7 @@ Scenario:  Healthcare worker can remove Single Contact Preference
     """
     * method patch
     * status 200
-    * def extensions = response.extension[5].extension.length
-    * if (extensions > 2) { karate.fail('Single Contact Preference not been removed from /extension/5/extension/2.')}
+    * def extensionsIn5thExtension = response.extension[5].extension.length
+    * match extensionsIn5thExtension == 2
+    * if (extensionsIn5thExtension > 2) { karate.fail('Single Contact Preference not been removed from /extension/5/extension/2.')}
```

Comment:
Not that there's anything wrong with what you've done, but more what I was trying to get at was checking the url of extension[5] (i'm not sure on the syntax, but something like this?)
`match responseextension[5].url == 'https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-NHSCommunication'`
and then again for the url of the next two extensions
`match responseextension[5].extension[0] == 'language'
match responseextension[5].extension[1] == 'interpreterRequired'`

Reply:
I have added a url checks on extension 5 and on it's nested extensions

Reply:
looks good, thanks